File: 단계별로_풀어보기/4. 1차원 배열/6. 10813.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

N, M = map(int, input().split())

resultArr = [i + 1 for i in range(N)]

for idx in range(M):
  i, j = map(int, input().split())
  resultArr[i - 1], resultArr[j - 1] = resultArr[j - 1], resultArr[i - 1]

output(f"{" ".join(str(x) for x in resultArr)}")

File: 단계별로_풀어보기/4. 1차원 배열/6. 10813.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString().split('\n')

const [N, M] = inputs[0].split(' ').map(val => +val)

const resultArr = Array(N)
  .fill(0)
  .map((_, idx) => idx + 1)

for (let idx = 1; idx <= M; idx++) {
  const [i, j] = inputs[idx].split(' ').map(val => +val)

  const temp = resultArr[i - 1]
  resultArr[i - 1] = resultArr[j - 1]
  resultArr[j - 1] = temp
}

process.stdout.write(`${resultArr.join(' ')}`)
